ferretdb_packaging/defineversion/docker_tags_test.go

```diff
@@ -0,0 +1,508 @@
+				"INPUT_PG_VERSION":  "16",
+			},
+			expected: &images{
+				developmentImages: []string{
+					"ghcr.io/otherorg/postgres-otherrepo-dev:ferretdb",
+				},
+			},
+		},
+
+		"push/main": {
+			env: map[string]string{
+				"GITHUB_BASE_REF":   "",
+				"GITHUB_EVENT_NAME": "push",
+				"GITHUB_HEAD_REF":   "",
+				"GITHUB_REF_NAME":   "main",
+				"GITHUB_REF_TYPE":   "branch",
+				"GITHUB_REPOSITORY": "FerretDB/documentdb",
+				"INPUT_PG_VERSION":  "16",
+			},
+		},
+		"push/main-other": {
+			env: map[string]string{
+				"GITHUB_BASE_REF":   "",
+				"GITHUB_EVENT_NAME": "push",
+				"GITHUB_HEAD_REF":   "",
+				"GITHUB_REF_NAME":   "main",
+				"GITHUB_REF_TYPE":   "branch",
+				"GITHUB_REPOSITORY": "FerretDB/documentdb",
+				"INPUT_PG_VERSION":  "16",
+			},
+		},
+
+		"push/tag/release": {
+			env: map[string]string{
+				"GITHUB_BASE_REF":   "",
+				"GITHUB_EVENT_NAME": "push",
+				"GITHUB_HEAD_REF":   "",
+				"GITHUB_REF_NAME":   "v0.102.0-ferretdb",
+				"GITHUB_REF_TYPE":   "tag",
+				"GITHUB_REPOSITORY": "FerretDB/documentdb",
+				"INPUT_PG_VERSION":  "16",
+			},
+			expected: &images{
+				developmentImages: []string{
+					"ferretdb/postgres-documentdb-dev:16-0.102.0-ferretdb",
+					"ferretdb/postgres-documentdb-dev:latest",
+					"ghcr.io/ferretdb/postgres-documentdb-dev:16-0.102.0-ferretdb",
+					"ghcr.io/ferretdb/postgres-documentdb-dev:latest",
+					"quay.io/ferretdb/postgres-documentdb-dev:16-0.102.0-ferretdb",
+					"quay.io/ferretdb/postgres-documentdb-dev:latest",
+				},
+				productionImages: []string{
+					"ferretdb/postgres-documentdb:16-0.102.0-ferretdb",
+					"ferretdb/postgres-documentdb:latest",
+					"ghcr.io/ferretdb/postgres-documentdb:16-0.102.0-ferretdb",
+					"ghcr.io/ferretdb/postgres-documentdb:latest",
+					"quay.io/ferretdb/postgres-documentdb:16-0.102.0-ferretdb",
+					"quay.io/ferretdb/postgres-documentdb:latest",
+				},
+			},
+		},
+		"push/tag/release-other": {
+			env: map[string]string{
+				"GITHUB_BASE_REF":   "",
+				"GITHUB_EVENT_NAME": "push",
+				"GITHUB_HEAD_REF":   "",
+				"GITHUB_REF_NAME":   "v0.102.0-ferretdb",
+				"GITHUB_REF_TYPE":   "tag",
+				"GITHUB_REPOSITORY": "OtherOrg/OtherRepo",
+				"INPUT_PG_VERSION":  "16",
+			},
+			expected: &images{
+				developmentImages: []string{
+					"ghcr.io/otherorg/postgres-otherrepo-dev:16-0.102.0-ferretdb",
+					"ghcr.io/otherorg/postgres-otherrepo-dev:latest",
+				},
+				productionImages: []string{
+					"ghcr.io/otherorg/postgres-otherrepo:16-0.102.0-ferretdb",
+					"ghcr.io/otherorg/postgres-otherrepo:latest",
+				},
+			},
+		},
+
+		"push/tag/release-rc-major-minor": {
+			env: map[string]string{
+				"GITHUB_BASE_REF":   "",
+				"GITHUB_EVENT_NAME": "push",
+				"GITHUB_HEAD_REF":   "",
+				"GITHUB_REF_NAME":   "v0.102.0-ferretdb-2.0.0-rc2",
+				"GITHUB_REF_TYPE":   "tag",
+				"GITHUB_REPOSITORY": "FerretDB/documentdb",
+				"INPUT_PG_VERSION":  "16.7", // set major and minor version
```

Comment:
If we set PG version with minor like `16.7`, it creates tags
`16-0.102.0-ferretdb-2.0.0-rc2`
`16.7-0.102.0-ferretdb-2.0.0-rc2`
`latest`.

If we set PG version without minor like `16` in test case `push/tag/release-major`, tags created are 
`16-0.102.0-ferretdb-2.0.0-rc2` 
`latest`.

Do we need code for PG version with minor, I'm wondering if that's just extra thing we can remove.

Reply:
So the input should be like [15.12, 16.8, 17.4] to include minor in this right https://github.com/FerretDB/documentdb/pull/16#discussion_r1974991174?